src/clj/game/cards/ice.clj

```diff
@@ -1060,6 +1060,12 @@
    "Rainbow"
    {:subroutines [end-the-run]}
 
+   "Ravana 1.0"
+   {:subroutines [{:label "Resolve a subroutine on another piece of rezzed bioroid ice."
```

Comment:
For consistency almost all messages use "ICE" (although in fairness, to match the printed rules and card texts, it probably should be all lowercase). You can drop the period at the end, too.